frontend/src/components/Collections/index.tsx

```diff
@@ -15,22 +15,16 @@ const Collections: FC = () => {
   return (
     <>
       <TitleWrapper>
-        <h1>Datasets</h1>
+        <TitleAndDescription>
+          <h1 data-test-id="collections-header">Collections</h1>
+          <p>Explore public collections of datasets or create your own.</p>
+        </TitleAndDescription>
         <CreateCollection />
       </TitleWrapper>
-      <p>
-        The cellxgene data portal is a repository of public, explorable
-        single-cell datasets. If you have a public dataset that you would like
-        hosted on this site, please let us know at{" "}
-        <a href="mailto:cellxgene@chanzuckerberg.com">
-          cellxgene@chanzuckerberg.com
-        </a>
-        .
-      </p>
-      <Heading />
-      {collections?.map(({ id, visibility }) => (
-        <Collection id={id} visibility={visibility} key={id} />
-      ))}
+      <CollectionsGrid
+        collections={collections}
```

Comment:
Hmm personally I prefer to do the filtering at this level, since it's business logic instead of `<CollectionsGrid />`'s own reusable component logic. This way the shared component is free of business logic as much as possible!

So something like:

```ts
<CollectionGrid
  collections={publicOnlyCollections(collections)}
/>
```

Reply:
I had the `CollectionsGrid` own visibility filtering because the component already passes its `accessType` prop to `CollectionRow`  since it owns the access type filtering as access type is only accessible after fetching the collection details.  I believe we're also planning on adding user filters/sorting to the grids in the future, so I thought leaving this modularity accessible would be beneficial

Reply:
Ahh got it! That sounds good 👍  I forgot about the broader context of the component 😂 Thanks for the reminder!